tiflash: add dttool docs (#7380)
pingcap/docs: tiflash/tiflash-command-line-flags.md

    - `--frame`: The size of the validation frame. The default value is `1048576`. This parameter is effective only when `version` is `2`.
    - `--compression`: The target compression algorithm. The value options are `lz4` (default), `lz4hc`, `zstd`, and `none`.
    - `--level`: The target compression level. The default value is `-1`, which means automatic mode. The value range varies with different compression algorithms.
    - `--config-file`: The configuration file of `dttool migrate` is the same as the [configuration file of `server`](/tiflash/tiflash-command-line-flags.md#server---config-file). When you use the configuration file, exit the local TiFlash server instance. For more information, see `--imitative`.
    - `--file-id`: The ID of the DTFile. For example, the ID of the DTFile `dmf_123` is `123`.
    - `--workdir`: The parent directory of `dmf_xxx`.
    - `--dry`: The dry run mode. Only the migration process is output.
    - `--nokeep`: Does not keep the original data. When this option is not enabled, `dmf_xxx.old` files are created.

> **Warning:**
>

> > TiFlash can read DTFile that uses custom compression algorithms and compression levels. However, only the `lz4` algorithm of the default compression level is officially supported. Custom compression parameters have not been thoroughly tested and are only experimental.

Reviewer: ```suggestion
> TiFlash can read DTFile that uses custom compression algorithms and compression levels. However, only the `lz4` algorithm with the default compression level is officially supported. Custom compression parameters have not been thoroughly tested and are only experimental.
```

> **Note:**
>

> > For security reasons, DTTool attempts to add lock to the working directory in the migration mode. Therefore, in the same directory at the same time, only one DTTool can perform a migration task. If you forcibly stop DTTool where the lock is not released, when you try to rerun DTTool later, DTTool might refuse to perform the migration task.

Reviewer: ```suggestion
> For security reasons, DTTool attempts to add a lock to the working directory in the migration mode. Therefore, in the same directory, only one DTTool can perform the migration task at the same time. If you forcibly stop DTTool where the lock is not released, then when you try to rerun DTTool later, it might refuse to perform the migration task.
```

>

> > If you encounter this situation, while ensuring data security, you can manually delete the LOCK file in the working directory to release the lock.

Reviewer: ```suggestion
> If you encounter this situation, you can manually delete the LOCK file in the working directory to release the lock under the circumstances that you are aware that the removal will not cause any data corruption.
```

## `dttool bench`

- Provides a basic I/O speed test for the DTFile.
- Parameters:

    - `--version`: The version of DTFile. See [`--version` in `dttool migrate`](#dttool-migrate).
    - `--algorithm`: The hash algorithm used for data validation. See [`--algorithm` in `dttool migrate`](#dttool-migrate).
    - `--frame`: The size of the validation frame. See [`--frame` in `dttool migrate`](#dttool-migrate).
    - `--column`: The columns of the table to be tested. The default value is `100`.
    - `--size`: The rows of the table to be tested. The default value is `1000`.
    - `--field`: The field length limit of the table to be tested. The default value is `1024`.
    - `--random`: The random seed. If you do not specify this parameter, the random seed is drawn from the system entropy pool.

>     - `encryption`: Enables the encryption feature.

Reviewer: ```suggestion
    - `--encryption`: Enables the encryption feature.
```
